app/views/catalog/_index_list_default.html.erb

```diff
@@ -25,8 +25,20 @@
   </div>
 <% end %>
 <div class="resource-type col-md-offset-2 col-md-10">
-  <%= render 'index_type_icon', document: document unless document.type_label.first.empty? %>
-  <%= doc_presenter.field_value index_fields(document)['type_label_tesim'] %>
+  <div class="col-md-4">
+    <%= render 'index_type_icon', document: document unless document.type_label.first.empty? %>
+    <%= doc_presenter.field_value index_fields(document)['type_label_tesim'] %>
+  </div>
+  <% if document.parents? %>
+    <div class="col-md-4">
+      Parent works:
```

Comment:
I think it would be nice to move the header text `Parent works:` to config/locales/hyrax.en.yml maybe under `hyrax` > `base`>`parents`>`header` for something like `<%= t("hyrax.base.parents.header") %>`